Makefile.main

```diff
@@ -199,10 +199,6 @@ docker-build: $(COMMANDS)
 	done;
 
 docker-push: docker-login docker-build
-	@if [ "$(BRANCH)" == "master" ]; then \
```

Comment:
this shouldn't be by default, we don't want to push in master for all the project. Should be a variable environment to turn on this capability.

Reply:
Added `DOCKER_PUSH_MASTER`